src/fee_lab_to_nwb/happ_ecephys/happmotifinterface.py

```diff
@@ -57,23 +61,75 @@ def get_synchronized_motif_timestamps(self):
 
         return motif_timestamps
 
+    def create_hierarchical_table_from_motif_timestamps(self, motif_timestamps: ArrayType):
+        """Create a hierarchical table from the timings of motifs.
+        The lowest hierarchical level is the level of syllables. The number of
+        syllables within a motif is assumed to be fixed per experiment. The latency
+        of each syllable is fixed for now, but should be modified in the future given
+        more information about the mapping between motifs and syllables."""
+        syllables = TimeIntervals(name="Syllables", description="desc")
```

Comment:
Ah I see what you mean, I think that should be possible, but currently I'm not doing it that way, just create a `TimeIntervals` object for the syllables. But I'll try it that way!